EIPS/eip-4635.md

```diff
@@ -0,0 +1,312 @@
+
+**Smart contracts MUST implement the ERC-165 `supportsInterface` function and signify support for the `IERC4635Receiver` interface to accept transfers. See "IERC4635Receiver ERC-165 rules" for further detail.**
+```solidity
+pragma solidity ^0.8.0;
+
+interface IERC4635Receiver {
+    /**
+        @notice Handle the receipt of a single ERCSFT token type.
+        @dev An ERCSFT-compliant smart contract MUST call this function on the token recipient contract, at the end of a `safeTransferFrom` after the balance has been updated.        
+        This function MUST return `bytes4(keccak256("onERCSFTReceived(address,address,uint256,uint256,bytes)"))` (i.e. 0x198fb191) if it accepts the transfer.
+        This function MUST revert if it rejects the transfer.
+        Return of any other value than the prescribed keccak256 generated value MUST result in the transaction being reverted by the caller.
+        @param _operator  The address which initiated the transfer (i.e. msg.sender)
+        @param _from      The address which previously owned the token
+        @param _type      The type of the token being transferred
+        @param _value     The amount of tokens being transferred
+        @param _data      Additional data with no specified format
+        @return           `bytes4(keccak256("onERCSFTReceived(address,address,uint256,uint256,bytes)"))`
+    */
+    function onERCSFTReceived(address _operator, address _from, uint256 _type, uint256 _value, bytes calldata _data) external returns(bytes4);
+
+    /**
+        @notice Handle the receipt of multiple ERCSFT token types.
+        @dev An ERCSFT-compliant smart contract MUST call this function on the token recipient contract, at the end of a `safeBatchTransferFrom` after the balances have been updated.        
+        This function MUST return `bytes4(keccak256("onERCSFTBatchReceived(address,address,uint256[],uint256[],bytes)"))` (i.e. 0xa3a64bf1) if it accepts the transfer(s).
+        This function MUST revert if it rejects the transfer(s).
+        Return of any other value than the prescribed keccak256 generated value MUST result in the transaction being reverted by the caller.
+        @param _operator  The address which initiated the batch transfer (i.e. msg.sender)
+        @param _from      The address which previously owned the token
+        @param _types     An array containing token types being transferred (order and length must match _values array)
+        @param _values    An array containing amounts of each token being transferred (order and length must match _ids array)
+        @param _data      Additional data with no specified format
+        @return           `bytes4(keccak256("onERCSFTBatchReceived(address,address,uint256[],uint256[],bytes)"))`
+    */
+    function onERCSFTBatchReceived(address _operator, address _from, uint256[] calldata _types, uint256[] calldata _values, bytes calldata _data) external returns(bytes4);       
```

Comment:
replace with number

Reply:
I've added the number 0x198fb191 in "This function MUST return `bytes4(keccak256("onERCSFTReceived(address,address,uint256,uint256,bytes)"))` (i.e. 0x198fb191) if it accepts the transfer." and 0xa3a64bf1 in "This function MUST return `bytes4(keccak256("onERCSFTBatchReceived(address,address,uint256[],uint256[],bytes)"))` (i.e. 0xa3a64bf1) if it accepts the transfer(s)." I will also add the number in the return